Formula/lua.rb

```diff
@@ -45,7 +45,8 @@ class Lua < Formula
   # sigaction provided by posix signalling power patch
   if build.with? "sigaction"
     patch do
-      url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      # url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      url "https://gist.githubusercontent.com/anonymous/99614c27bcac2aa86181fabe78c85915/raw/7abdf0234ffd38fc9094078d8128938f5e2accd4/lua-5.2.3-sig_catch.patch"
```

Comment:
To be fair, all commit discussions are pinged onto the team chat, so it's not like it was intentionally buried, and the change is so incredibly minor in terms of consequences; URLs still work, existing clones still work, and you can rename it to something else or rename it back to what it was pretty much endlessly without issue.

Short of deleting the entire repo, or force-pushing over it with emptiness, it's really pretty hard to break patches for the current use case.